netbout-client/pom.xml

```diff
@@ -180,7 +180,12 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911 Fix findbugs errors in client
```

Comment:
@exper0 it contains 17 words
Rultor will reject puzzles that less than 20 words